templates/CONTRIBUTING.md

```diff
@@ -16,148 +16,216 @@ By contributing to this repository, you agree:
-    - `Requirements`: a bullet point list of the requirements for each task, including links to each requirement page. Requirements are numbered by `RXXXXX`.
-- The persona, objective, user tasks, and requirements need to be linked to their corresponding description page (detailed in the following sections)
-- Save use case files in the [use-cases directory](https://github.com/nih-cfde/usecases/tree/master/docs/use-cases)
-
-### `Persona` files
-- Use the [persona file template](https://github.com/nih-cfde/usecases/tree/master/docs/template_files/persona-template.md) to enter the required sections
+    - yaml index header, including keys for `title:`, `persona:`, `objective:`, `user_tasks:`, and `requirements:`
+    - a short summary of the use case
+
+#### `Persona` files <a name="persona-files"></a>
+- Use the [persona file template](https://github.com/nih-cfde/use-case-library-build/tree/latest/library_file_templates/library-persona-template.md) to enter the required sections
 - Required sections:
-    - yaml index header
-    - Short description of their biological/computational experience, their role and responsibilities, and relation to any of the other Personas in the Use Cases Repository
-    - A section listing assumptions about the persona's credentials e.g., access to the CFDE
-- Save persona files in the [personas directory](https://github.com/nih-cfde/usecases/tree/master/docs/personas/)
+    - yaml index header with `title:`
+    - Short description of the persona's biological/computational experience, their role and responsibilities, and relation to any of the other personas in the Use Case Library
+    - A bullet point section listing assumptions about the persona's credentials e.g., access to the CFDE
 
-### `Objective` files
-- Use the [objective file template](https://github.com/nih-cfde/usecases/tree/master/docs/template_files/objective-template.md) to enter the required sections
+#### `Objective` files <a name="obj-files"></a>
+- Use the [objective file template](https://github.com/nih-cfde/use-case-library-build/tree/latest/library_file_templates/library-obj-template.md) to enter the required sections
 - Required sections:
-    - yaml index header
-    - `<name of the task>`: brief ~1 sentence description of the task
-- Save objective files in the [objectives directory](https://github.com/nih-cfde/usecases/tree/master/docs/objectives/)
-
-### `User task` files
-- The User Tasks title should be short and self-explanatory, but please add a short description as needed for clarity
-- User Task titles always begin with their `TXXXX` ID
-- Use the [user task file template](https://github.com/nih-cfde/usecases/tree/master/docs/template_files/user-task-template.md) to enter the required sections
+    - yaml index header with `title:`
+    - a brief ~1-2 sentence description of the objective
+
+#### `User task` files <a name="user-task-files"></a>
```

Comment:
the requirements here have to match the use case.

Reply:
hmm. that change isn't quite right. the use case should only have the total of requirements of it's tasks, but you can't put all the requirements from a use case into each task.

Reply:
right, yes struggling to make the wording convey that. will edit!

Reply:
that seems right i think